src/static_handler.cc

```diff
@@ -0,0 +1,77 @@
+#include "static_handler.h"
+#include "fileLoader.h"
+#include <iostream>
+#include <boost/filesystem.hpp>
+
+using std::string;
+size_t STATEMENT_SIZE = 2;
+
+RequestHandler::Status StaticHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+  path_ = uri_prefix;
+  for(size_t i = 0; i < config.statements_.size(); i++) {
+    if (config.statements_[i]->tokens_.size() == STATEMENT_SIZE){
+        if(config.statements_[i]->tokens_[0] == "root"){
+          filebase_ = config.statements_[i]->tokens_[1];
+        }
+        else{
+          std::cerr << "Invalid Static Handler 'filebase' parameter." << std::endl;
+          return RequestHandler::Error;
+        }
+    }
+    else{
+      std::cerr << "Static Handler must specify 'filebase' parameter." << std::endl;
```

Comment:
I think this error message is inaccurate? Maybe should say "invalid Static Handler config" or something